ui/component/viewers/videoViewer/view.jsx

```diff
@@ -133,16 +124,16 @@ function VideoViewer(props: Props) {
         currentVideo.removeEventListener('volumechange', doVolume);
       }
     };
-  }, []);
+  }, [changeMute, changeVolume, onEndedCB]);
```

Comment:
What does this fix?

Reply:
It was an autosuggested fix from the dependencies..maybe I was too trigger happy :)